src/main/java/org/gradlex/javaecosystem/capabilities/rules/AopallianceRule.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 the GradleX team.
```

Comment:
Happy new year 😉

```suggestion
 * Copyright 2023 the GradleX team.
```

Reply:
This fails the license header check. Lets fix this separately.